strawberryfields/program.py

```diff
@@ -574,3 +577,52 @@ def draw_circuit(self, tex_dir='./circuit_tex', write_to_file=True):
             document = drawer.compile_document(tex_dir=tex_dir)
 
         return [document, tex]
+
+
+    def args(self, *args):
```

Comment:
Maybe this isn't a great name for the method? Especially since the argument has the same name.

Reply:
Feel like ``params`` is probably the best name. That's what we usually use in pennylane, and it ties to "parameterized circuits" as well

Reply:
I'll change it.